day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,32 @@
+f = (1..100).to_a
```

Comment:
Nice fizzbuzz Carleigh! This is a solid implementation, because it works! Can you think of a way to break up the concerns into methods or a class? How about accepting user input? If you have time this weekend, then I encourage you to time box each of the other day-7 challenges to a few hours and give them a shot. This will be great practice for Mod-1 by giving yourself a time constraint, and it will ensure you don't spend too much time on them. Overall great job on the capstone and in Mod-0 Carleigh!